app/models/profile.rb

```diff
@@ -191,6 +191,21 @@ def deactivate(reason)
     update!(active: false, deactivation_reason: reason)
   end
 
+  # Update the profile's deactivation reason to "encryption_error". As a
+  # side-effect, this will also update the associated pending in-person
+  # enrollment to have a status of "cancelled".
+  def deactivate_due_to_encryption_error
+    update!(
+      active: false,
+      deactivation_reason: :encryption_error,
+      in_person_verification_pending_at: nil,
```

Comment:
It looks like we are not clearing other timestamps on deactivation, so I would say we shouldn't clear `in_person_verification_pending_at`